src/CMakeLists.txt~

```diff
@@ -0,0 +1,128 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Ah, now I see, on my branch I have CMakeLists.txt and CMakeLists.txt~, which is a bad copy, I guess :)